test/identical.js

```diff
@@ -25,5 +26,37 @@ describe('identical', function() {
     eq(R.identical(new Number(0), 0), false);
     eq(R.identical(new Number(0), new Number(0)), false);
   });
+  
+  it('is auto-curried', function() {
+    assert.strictEqual(R.identical.length, 2);
+    var unaryFn = R.identical("foo");
+    assert.strictEqual(unaryFn.length, 1);
+    eq(unaryFn("bar"), false);
+    eq(unaryFn("foo"), true);
+    
+    eq(R.identical()("foo")()("foo"), true);
```

Comment:
Do we need to check the functions arity? Wouldn't these two tests be enough:

```javascript
eq(R.identical("foo", "foo"), true);
eq(R.identical("foo")("foo"), true);
```

The fact that things such as `add()(1)()(2)` work is just a consequence of how currying is internally implemented AFAIK. I'm not sure we need to showcase this use case or make sure it remains supported. OTOH this could be seen as a breaking change if you opted out from it so up to you ;)

Reply:
Removed the line with `()` fake calls.